contrib/ivorysql_ora/README.md

```
@@ -1,7 +1,112 @@
+make -C contrib/ivorysql_ora installcheck
+```
+
+This executes all test cases listed in `ORA_REGRESS` (character types,
+datetime, intervals, numeric types, binary float/double, raw/long, built-in
+functions, MERGE, system views, LIKE operator, XML functions, DBMS_OUTPUT,
+DBMS_LOCK, DBMS_UTILITY, DBMS_SESSION, UTL_FILE, UTL_RAW, UTL_ENCODE,
+STRAGG, and ASCII handling).
+
+## Directory Layout
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)